### PR TITLE
Cloud Messages docs

### PR DESCRIPTION
**Pull request by donnyyung**
We added a new key to the `cloud` field of `config.yml` so let's update docs to reflect that.

### CODE REVIEW

File: docs/v2.4/reference/config.md [hunk 1]
```diff
@@ -70,12 +71,13 @@ These are the valid fields for the `images` key:
...
 These fields control how the client interacts with the Cloud service.
-Currently there is only one key and it accepts bools: `1`, `t`, `T`, `TRUE`, `true`, `True`, `0`, `f`, `F,` `FALSE`
 
-|Field|Description|Default|
-|---|---|---|
-|`skipLogin`|Whether the cli should skip automatic login to Ambassador Cloud. If set to true, you must have a [license](../cluster-config/#air-gapped-cluster) installed in the cluster in order to be able to perform selective intercepts |false|
+|Field|Description|Type|Default|
+|---|---|---|---|
+|`skipLogin`|Whether the cli should skip automatic login to Ambassador Cloud. If set to true, you must have a [license](../cluster-config/#air-gapped-cluster) installed in the cluster in order to be able to perform selective intercepts |bools: `1`, `t`, `T`, `TRUE`, `true`, `True`, `0`, `f`, `F,` `FALSE`|false|
+|`refreshMessages`|How frequently the CLI should communicate with Ambassador Cloud to get new command messages, which also resets whether the message has been raised or not. So you will see at most once within the duration given by this config|duration: number respresenting seconds or a string with a unit suffix of `ms`, `s`, `m`, or `h`|168h|
```
Comment (josecv) [inline docs/v2.4/reference/config.md:80]: `So you will see *each message* at most once within...`

File: docs/v2.4/reference/config.md [hunk 2]
```diff
@@ -70,12 +71,13 @@ These are the valid fields for the `images` key:
 |`webhookAgentImage`|The container image that the [Traffic Manager](../cluster-config/#mutating-webhook) will use when installing the Traffic Agent in annotated pods *This value is only used if a new traffic-manager is deployed*||
 
 #### Cloud
+Values for `clouds` are listed below and their type varies, so please see the chart for the expected type for each config value.
```
Comment (josecv) [inline docs/v2.4/reference/config.md:74]: I think this should read "Values for `cloud`"

File: docs/v2.4/reference/config.md [hunk 3]
```diff
@@ -22,8 +23,8 @@ timeouts:
 logLevels:
   userDaemon: debug
 images:
-  registry: privateRepo
-  agentImage: ambassador-telepresence-agent:1.8.0
+  registry: privateRepo # This overrides the default docker.io/datawire repo
+  agentImage: ambassador-telepresence-agent:1.8.0 # This overrides the agent image to inject when intercepting
 grpc:
   maxReceiveSize: 10Mi
```
Comment (josecv) [inline docs/v2.4/reference/config.md:29]: should this example config maybe include a `cloud:` section? I have no strong feelings on it either way, cause i think the cloud config is specialized enough to warrant being skipped in the example, but wanna get your thoughts

**Review by josecv — CHANGES_REQUESTED** [review]
small wording issues

File: docs/v2.4/reference/config.md (hunk 3 above)
Comment (donnyyung) [reply docs/v2.4/reference/config.md:31]: Doesn't hurt, so I'll add it!

**Review by josecv — APPROVED** [review]

**Pull request merged**